README.rst

```diff
@@ -12,6 +12,11 @@ used to generate the `official documentation`_.
 See the `contributor's guide <CONTRIBUTING.md>`_ for acceptable changes and
 instructions to build the documentation locally.
 
+You can use the API stubs in this repository for syntax highlighting when
```

Comment:
The subs don't provide syntax highlighting, they provide code completion (intellesense).

Reply:
provides syntax highlighting and code completion.
w/o stubs unknown classes are shown in white, with stubs in green and also offer code completion. We could mention both.

Reply:
For me syntax highlighting, code completion and (life) type checking somehow belong together, so I did not mention in detail.

Reply:
Sure we can mention both.